CardScan/Classes/VideoFeed.swift

```diff
@@ -176,10 +176,24 @@ class VideoFeed {
         videoDevice.unlockForConfiguration()
     }
     
+    //MARK: -Torch Logic
     func toggleTorch() {
         self.torch?.toggle()
     }
     
+    func isTorchOn() -> Bool {
+        return self.torch?.state ==  Torch.State.on ? true : false
```

Comment:
Isn't the `? :` redundant here?

Reply:
well isnt the state an enum and not a bool? so if its not on, return False

Reply:
yeah, you need the `self.torch?.state == Torch.state.on` part, just not the  `? true : false` part

Reply:
OOOOO yeap yeap yeap youre right